test/unit/mp_error.result

```diff
@@ -58,6 +58,8 @@ ok 5 - subtests
     ok 9 - key double
     ok 10 - key bool
     ok 11 - key uuid
+    ok 12 - line 515
+    ok 13 - line 516
```

Comment:
All the cases in the test pass the extra string. Let's add "key mp" and "key mp size" descriptions for consistency.

Also, while we are at it, let's drop the result file if we can (in a separate commit).

Reply:
Fixed. Result file is removed in a separate commit.

```diff
@@ -512,8 +514,9 @@ test_payload(void)
           tt_uuid_is_equal(&uuid, &val_uuid), "key uuid");
        uint32_t val_mp_size;
        const char *val_mp = error_get_mp(e, "key7", &val_mp_size);
-       ok(val_mp_size == mp_size);
-       ok(val_mp != NULL && memcmp(val_mp, mp, mp_size) == 0);
+       ok(val_mp_size == mp_size, "key msgpack size");
+       ok(val_mp != NULL &&
+          memcmp(val_mp, mp, mp_size) == 0, "key msgpack data");

        error_unref(e);
```